test/cat.jl

```diff
@@ -51,23 +51,25 @@ module TestCat
 
     @testset "hcat ::AbstractVectors" begin
         df = DataFrame()
-        DataFrames.hcat!(df, CategoricalVector{Union{Int, Missing}}(1:10))
+        DataFrames.hcat!(df, CategoricalVector{Union{Int,Missing}}(1:10), makeunique=true)
         @test df[1] == CategoricalVector(1:10)
         DataFrames.hcat!(df, 1:10, makeunique=true)
         @test df[2] == collect(1:10)
         DataFrames.hcat!(df, collect(1:10), makeunique=true)
         @test df[3] == collect(1:10)
 
         df = DataFrame()
-        df2 = hcat(CategoricalVector{Union{Int, Missing}}(1:10), df)
```

Comment:
Can you revert all the renames in this file and the following files?

Reply:
Yes, don't worry, I'm not ignoring you, I was busy...

Reply:
No worries. I leave new comments because GitHub hides the old ones and it's hard to keep track.

Reply:
Yeah, I also never know if you are aware that I made changes, though by now it's seeming that you must get a notification whenever I commit...